.github/docker/centreon-web/bookworm/Dockerfile.dependencies

```diff
@@ -134,3 +134,9 @@ apt-get install -y \
 apt-get clean
 
 EOF
+
+###################
+# install plugins #
+###################
+
+RUN apt install -y centreon-plugin-operatingsystems-linux-snmp
```

Comment:
Integrate this command in the previous `RUN` instruction (before `apt-get clean`)